app/helpers/toolbar_helper.rb

```diff
@@ -0,0 +1,28 @@
+module ToolbarHelper
+  def toolbar(title:, subtitle: '', &_)
```

Comment:
see above: Why are you including the block parameter if you are not using it anyway?